src/model/PowerAuthClientConfiguration.ts

```diff
@@ -31,6 +31,34 @@ export interface PowerAuthClientConfigurationType {
      * If not provided, then the read timeout is set to 20 seconds.
      */
     readonly readTimeout?: number
+    /**
+     * Custom HTTP headers that will be added to each HTTP request produced by this library.
+     */
+    readonly customHttpHeaders?: PowerAuthHttpHeader[] | null;
+    /**
+     * Basic HTTP Authentication that will be added to each HTTP request produced by this library.
+     */
+    readonly basicHttpAuthentication?: PowerAuthBasicHttpAuthentication | null;
+}
+
+/**
+ * Custom HTTP header data.
+ */
+export interface PowerAuthHttpHeader {
+    /** Key of the HTTP header. */
+    key: string
+    /** Value of the HTTP header. */
+    value: string
```

Comment:
Please rename this to more web friendly naming:
- `headerName`
- `headerValue`

Or simpler:
- `name`
- `value`

This will be also beneficial in future, when we decide to support headers with random generated value.

Reply:
Yop, makes sense